lib/xapi/problem.rb

```diff
@@ -79,5 +75,12 @@ def path(file_name)
     def metadata
       @metadata ||= YAML.load(File.read(yaml))
     end
+
+    def find_file(default_basename, suffix)
```

Comment:
In terms of naming:

- the method name should be named for what it returns, which is a `path` or `path_to`.
- The `default_basename` is true only in one of the cases. How about `target`?
- While suffix is technically correct, `extension` would be more precise.